### DISCUSSION TITLE
VcImageryProviderWmts设置tilingScheme属性

### DISCUSSION THREAD

**Discussion by JennicaLi** [zouyaoji/vue-cesium#385] [Q&A]
VcImageryProviderWmts设置tilingScheme属性该在哪里初始化呀，直接在setup设置获取不到Cesium

**Comment by zouyaoji** [comment]
这种一般有2个方案，
（1）先定义一个空的变量，等vc-viewer的cesiumReady或者viewerReady事件后再来初始化这个变量，这时候就有Cesium了。子组件可以写个v-if你的变量不为空了再初始化。像 vue-cesium-demo中的子组件都是等vc-viewer ready之后才挂载的。
（2）直接把Cesium.js在 index.html中引入，Cesium就直接是全局变量了，vue-cesium不会再引入cesium，setup中可以直接写。
